src/components/DiffView/index.tsx

```diff
@@ -265,6 +284,39 @@ export class DiffViewBase extends React.Component<Props> {
     return <div className={styles.extraMessages}>{messages}</div>;
   };
 
+  renderGutter = ({
+    change,
+    renderDefault,
+    side,
+    wrapInAnchor,
+  }: RenderGutterParams) => {
+    const { enableCommenting, version } = this.props;
+
+    const { isInsert, lineNumber } = change;
+    const changeKey = getChangeKey(change);
+
+    let gutter = wrapInAnchor(renderDefault());
+    if (enableCommenting && isInsert && lineNumber && side === 'new') {
+      gutter = (
+        <div className={styles.gutter}>
+          {gutter}
+          <Commentable
```

Comment:
This approach seems to work but it would be nicer if one could hover over a line (not just the gutter) and see the comment button appear. This would convey that it's possible to add comments since that may not be obvious. 

To do that, we'd need to replace [the whole `<tr>`](https://github.com/otakustay/react-diff-view/blob/7a97efa2b1ed11c79cd755ba2fb0475de82b09a7/src/Hunk/UnifiedHunk/UnifiedChange.js#L80) with `<Commentable as="tr" />`. I don't think that's possible but it might be worth filing a feature request for it. They are super fast about fixing things.

But, yeah, the `renderGutter()` approach still works and I think it's worth pursuing that as a quick fix.

Reply:
Good suggestion. I opened https://github.com/otakustay/react-diff-view/issues/70.

Reply:
@bobsilverberg thanks for opening an issue. Rather than "wrap" the `<tr>` we really want to provider our own `<tr>` tag with custom props, like a class name. Maybe that's obvious enough in your issue description but I wanted to point it out since it's not possible to wrap `<tr>` tags, it will break the table layout.

Reply:
He's given me some suggestions of how to make this work. I think I'm going to try to land it first the way it is, with the button in the gutter, as we're still going to need that even if I also use `renderToken` to allow for hovering over the code area. I will file a follow-up to look into adding `renderToken` as well.